src/config/column-generator.js

```diff
@@ -70,9 +74,24 @@ const renderLinkCells = (linkObjs) => {
 
 const renderDateCell = ({
   iso8601Date,
-}) => (
-  <CellDiv>{moment(iso8601Date).format(DATE_FORMAT)}</CellDiv>
-);
+}) => {
+  const {
+    relativeDates,
+  } = useSelector((state) => state.settings);
+  if (!iso8601Date) {
+    return <CellDiv>--</CellDiv>;
+  }
+  if (relativeDates) {
+    return (
+      <CellDiv>
+        <Tooltip label={moment(iso8601Date).format(DATE_FORMAT)}>
+          {moment(iso8601Date).fromNow()}
```

Comment:
Instead of `fromNow()`, could we do `fromNow(true)` so that we get Age as "35 minutes" rather than "35 minutes ago"?

Reply:
Adjusted that now. The Age column won't show the suffix. If you enable Relative Dates, the Created At, Updated At, Last Status Update and Last Note Added columns will all show the suffix.